treeherder/webapp/api/jobs.py

```diff
@@ -54,22 +54,37 @@ def retrieve(self, request, project, jm, pk=None):
     def list(self, request, project, jm):
         """
         GET method implementation for list view
-
+        Optional paramters (default):
+        - offset (0)
+        - count (10)
+        - full (false)
```

Comment:
isn't the ``full`` param removed now?  We could remove it from this list of optional fields.